packages/async/src/babel-plugin.ts

```diff
@@ -2,6 +2,7 @@ import Types from '@babel/types';
 import {NodePath, Binding} from '@babel/traverse';
 
 const DEFAULT_PACKAGES_TO_PROCESS = {
+  '@shopify/alpaql/async': ['createAsyncQuery'],
```

Comment:
Since `alpaql` is an external package to the `quilt` codebase, does it make more sense to add this into sewing-kit via the `packages` option?

Reply:
Agree, this took a while to find because I didnt expect @shopify/async to have these implicit connection to other packages. Ideally `react-async` and `@shopify/react-graphql` are move out as well, but since that will be a breaking change, we can leave it for now.

Reply:
Although we might have a problem splitting it out now since the logic below is OR
`(state.opts && state.opts.packages) || DEFAULT_PACKAGES_TO_PROCESS`

sk wil have to included all the `DEFAULT_PACKAGES_TO_PROCESS` in order to achieve backward compatibility.
so either we keep the `alpaql` logic here, or we make a breaking change for the babel-plugin options to be additive.

Reply:
I agree with Michelle https://github.com/Shopify/quilt/pull/1702#pullrequestreview-548682728

Yes, it should probably live in sewing-kit, and probably should be done when we wanna move out react-async and react-graphql as well.